### DISCUSSION TITLE
"Start bot" can take minutes to complete

### DISCUSSION THREAD

**Discussion by irwinwilliams** [microsoft/BotFramework-Composer#5591] [General]
So far in using Composer, whenever I press "Start bot", it takes a few minutes to be ready.
What's happening in the back there? Is there something I should do to speed that up?

**Comment by a-b-r-o-w-n** [comment]
There are multiple things that could be happening that make this slow. First, if your bot is using Luis for language understanding, we need to extract, upload and train the language models. Another reason this is  slow is that the bot project needs to be compiled. We are always looking to improve this experience.  

cc @boydc2014
